plugins/package-configuration-providers/dir/src/main/kotlin/DirPackageConfigurationProvider.kt

```diff
@@ -51,7 +51,7 @@ open class DirPackageConfigurationProviderFactory :
 
     override fun create(config: DirPackageConfigurationProviderConfig) = DirPackageConfigurationProvider(config)
 
-    override fun parseOptions(options: Options) =
+    override fun parseConfig(options: Options) =
```

Comment:
I basically agree with the change, but I believe the rationale "to name the function after the output instead of the input" isn't a good one, because "parse" refers to the input, not the the output, and it makes more sense to say what is being parsed than what the output will be. A better rationale IMO would be to do this as a preparation for the `options` vs `secrets` split, to find a generic term that fits both.